package.json

```diff
@@ -63,10 +63,11 @@
     "@storybook/addon-links": "^8.1.6",
     "@storybook/addon-onboarding": "^8.1.6",
     "@storybook/blocks": "^8.1.6",
+    "@storybook/builder-vite": "^8.1.10",
     "@storybook/react": "^8.1.6",
     "@storybook/react-vite": "^8.1.6",
     "@storybook/test": "^8.1.6",
-    "@storybook/builder-vite": "^8.1.10",
+    "@testing-library/jest-dom": "6.4.5",
```

Comment:
Why do we add fixed version `6.4.5` here - while `6.4.8` got published some weeks ago. Are there any issues with that?

Reply:
When I created the PR, the Automated GitHub Workflow Action `Check Dependencies / check-dependencies` failed due to the latest version of `@testing-library/jest-dom`. 

Please advise on how to proceed with directly modifying the dependencies.

Reply:
okay, updated your branch and it seems resolved now